src/databricks/labs/ucx/source_code/known.json

```diff
@@ -1977,6 +1977,228 @@
+      },
+      {
+        "code": "legacy-context-in-shared-clusters",
+        "message": "sparkContext is not supported on UC Shared Clusters. Rewrite it using spark"
+      },
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate 'self._spark.table(self._meta_ingest_tbl)' because its table name argument cannot be computed"
+      },
+      {
+        "code": "table-migrate",
+        "message": "The default format changed in Databricks Runtime 8.0, from Parquet to Delta"
+      }
+    ],
+    "spark_price_transparency.pt_functions": [],
+    "spark_price_transparency.pt_ingest_table": [
+      {
+        "code": "legacy-context-in-shared-clusters",
+        "message": "sparkContext is not supported on UC Shared Clusters. Rewrite it using spark"
+      }
+    ],
+    "spark_price_transparency.pt_raw": [
+      {
+        "code": "dbfs-usage",
+        "message": "Deprecated file system path: dbfs:/user/hive/warehouse/pt_raw.db"
+      }
+    ],
+    "spark_price_transparency.pt_schema": [
+      {
+        "code": "dbfs-usage",
+        "message": "Deprecated file system path: dbfs:/tmp/pt/_checkpoint"
+      }
+    ],
+    "spark_price_transparency.pt_stage": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate table_name argument in 'self._spark.sql(f'CREATE DATABASE IF NOT EXISTS {self.stage_db_name}')' because its value cannot be computed"
+      }
+    ],
+    "spark_price_transparency.pt_table": [
+      {
+        "code": "table-migrate",
+        "message": "SQL query is not supported yet: DELETE FROM  WHERE 1=1"
+      },
+      {
+        "code": "table-migrate",
+        "message": "SQL query is not supported yet: DROP TABLE IF EXISTS "
+      },
+      {
+        "code": "table-migrate",
+        "message": "The default format changed in Databricks Runtime 8.0, from Parquet to Delta"
+      }
+    ],
+    "spark_price_transparency.pt_types": [],
+    "spark_price_transparency.table": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate 'self.spark.table(self.wh_tbl_name)' because its table name argument cannot be computed"
+      },
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate table_name argument in 'self.spark.sql(f'DELETE FROM {self.wh_tbl_name} WHERE 1=1')' because its value cannot be computed"
+      },
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate table_name argument in 'self.spark.sql(f'DROP TABLE IF EXISTS {self.wh_tbl_name}')' because its value cannot be computed"
+      },
+      {
+        "code": "table-migrate",
+        "message": "The default format changed in Databricks Runtime 8.0, from Parquet to Delta"
+      }
+    ],
+    "spark_price_transparency.table_analytic": [
+      {
+        "code": "table-migrate",
+        "message": "SQL query is not supported yet: DELETE FROM  WHERE 1=1"
```

Comment:
noise. please change the error code for that

Reply:
done